### ISSUE TITLE
when using yolov4x-mish, got a negative iouloss value in training

### ISSUE THREAD

**Issue by zzzyy1997** [AlexeyAB/darknet#7640]
`v3 (mse loss, Normalizer: (iou: 0.75, obj: 4.00, cls: 0.50) Region 168 Avg (IOU: 0.607446), count: 5, class_loss = 168.960709, iou_loss = -124.604523, total_loss = 44.356182
v3 (mse loss, Normalizer: (iou: 0.75, obj: 1.00, cls: 0.50) Region 185 Avg (IOU: 0.600771), count: 25, class_loss = 18.781467, iou_loss = 10.009899, total_loss = 28.791367
v3 (mse loss, Normalizer: (iou: 0.75, obj: 0.40, cls: 0.50) Region 202 Avg (IOU: 0.499021), count: 10, class_loss = 2.189251, iou_loss = 3.335687, total_loss = 5.524939
 total_bbox = 531095, rewritten_bbox = 0.151385 %
v3 (mse loss, Normalizer: (iou: 0.75, obj: 4.00, cls: 0.50) Region 168 Avg (IOU: 0.739929), count: 5, class_loss = 98.715279, iou_loss = -73.244339, total_loss = 25.470942
v3 (mse loss, Normalizer: (iou: 0.75, obj: 1.00, cls: 0.50) Region 185 Avg (IOU: 0.568532), count: 22, class_loss = 17.216768, iou_loss = 11.470327, total_loss = 28.687096
v3 (mse loss, Normalizer: (iou: 0.75, obj: 0.40, cls: 0.50) Region 202 Avg (IOU: 0.383138), count: 8, class_loss = 1.523634, iou_loss = 2.342752, total_loss = 3.866386
 total_bbox = 531130, rewritten_bbox = 0.151375 %`

 CUDA-version: 10200 (10200), cuDNN: 7.6.5, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.5.2
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce Titan V
net.optimized_memory = 0
mini_batch = 2, batch = 64, time_steps = 1, train = 0

**Comment by ekesdf** [comment]
What dataset do you use bc it seems that there are some false labels in your dataset bc all other iou_loss are normal so.
Check your dataset and look for IMG with only background or very small labels.
And plz submit your config for further decision-making on what went wrong during the training

**Comment by zzzyy1997** [comment]
Sorry for the unclear explanation and thank for you reply.
i use a custom dataset, i will recheck my dataset and hope it will helpful.

**Comment by ekesdf** [comment]
np next time you will make it better :)
